docs/change-log/index.md

```diff
@@ -942,6 +967,7 @@ with a drill-down to view the steps and their details. [Tech Preview]
 
 | Will be removed|Deprecated|API                                                                                |Use instead                                                                                                                                                 |
 |---------------|------------|----------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| v1.9.0       |v1.6.3    |`FunctionSpec.clone_target_dir`                                                      |`ImageBuilder.source_target_dir`
```

Comment:
uncertain it is needed. correct @alonmr ?

Reply:
Users should no be using this directly but I think it's good to mention as they may want to change it manually in function yamls. @jillnogold please change `source_target_dir` to `source_code_target_dir`